_posts/2021-03-03-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: blog-detail
+post-type: blog
+by: Scala Center
+title: Developers Preview Survey Results
```

Comment:
I find “Developers Preview Survey” hard to grasp. I think “Scala 3” is missing somewhere?

```suggestion
title: Scala 3 Developer’s Preview Survey Results
```